project/Build.scala

```diff
@@ -1416,7 +1413,7 @@ object Build {
           val run = jsEnv.start(input, config)
           Await.result(run.future, Duration.Inf)
           Set(out)
-        } ((irFiles :+ linkerModule).toSet)
+        } ((cp :+ linkerModule).toSet)
```

Comment:
@sjrd I'm not sure whether this is a problem. Basically we pass directories instead of actual files now.

Reply:
The [doc of `FileFunction`](https://www.scala-sbt.org/1.0/api/sbt/util/FileFunction$.html) is not very clear about this, but I believe it needs an actual set of files, not directories.